src/vmm/src/guest_config/templates/mod.rs

```diff
@@ -545,17 +551,32 @@ mod tests {
     fn test_serialization_lifecycle() {
         #[cfg(target_arch = "x86_64")]
         {
+            let start = Instant::now();
             let cpu_config: X86_64CpuTemplate = serde_json::from_str(X86_64_TEMPLATE_JSON)
                 .expect("Failed to deserialize x86_64 CPU template.");
+            let end = Instant::now();
+            assert!(
+                end.duration_since(start).as_micros() < TEMPLATE_DESERIALIZATION_MAX_US,
```

Comment:
Do you believe this template is representative for the duration test? Would it be a good idea to have some range (maybe 2 or 3 templates of different lengths)?

Reply:
Sounds like a good idea.